### ISSUE TITLE
Image preprocessing

### ISSUE THREAD

**Issue by seann999** [deepmind/kinetics-i3d#13]
The readme says to scale the RGB values between -1 and 1. Does this mean `x/128.0-1.0`, where x is an uint8 image?
I'm more used to seeing normalizing images with mean and std, so I want to make sure.

**Comment by zekun-li** [comment]
I have the same question. However when I do `x/128. - 1.` on the sample data, the result doesn't match with the given `.npy`

**Comment by egg347** [comment]
@zekun-li Hi, the provided *.npy file has shape (1, num_frames, 224, 224, 3), I wonder what "1" refers to? And what's its value?

**Comment by zekun-li** [comment]
@egg347 That dimension corresponds to batch size.

**Comment by egg347** [comment]
@zekun-li Thanks a lot! Have you solved your problem above? It seems that the sample data firstly rescale to [-1,1](videos to videos),and secondly rescale to [0,1](videos to *.npy file).
 By the way, I'm confused about the output of the sample. According to the "evalutate_sample.py", it prints Norm of logits、out_predictions[index]、out_logits[index]、kinetics_classes[index])。So what Norm of logits and out_logits[index] can be used for?
With no offense, are you Chinese? Your id looks like a Chinese name. Would you please leave me your EMAIL  or your WECHAT id if you don't mind? It comforts me  to communicate in Chinese.

**Comment by TianjiPang** [comment]
@zekun-li I also have the same problem on preprocessing. No matter rescaling the image on R, G, B channel individually or across RGB channals, the results all can't match with the given .npy. Have you solved the problem?

**Comment by vivoutlaw** [comment]
@zekun-li @egg347 @TianjiPang @seann999 : Hi guys, were you able to resolve the scaling. If yes, would you mind sharing what you did. Thanks

**Comment by seann999** [comment]
I think I simply did what I first guessed, x/128.0-1.0.

**Comment by vivoutlaw** [comment]
@seann999: I did the same too, I hope the extracted feature are alright.

**Comment by sbelharbi** [comment]
hello,
any updates on this rgb normalization issue?
it is still not clear how rgb image is normalized.
normalizing with `(img/255.) * 2 - 1` or imagenet mean and std does not lead to normalized image that matches their provided `*.npy` at https://github.com/google-deepmind/kinetics-i3d/blob/master/data/v_CricketShot_g04_c01_rgb.npy
`np.abs(my_normed - original_normed.npy).mean()` gives:
0.01983683090857856  # ((img / 255.) * 2 - 1)
0.0205170704450589  # ((img / 128.) - 1)
0.26038003  # for imagenet normalization

thanks
